src/main/resources/META-INF/web-fragment.xml

```diff
@@ -0,0 +1,17 @@
+<web-fragment xmlns="http://java.sun.com/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-fragment_3_0.xsd" version="3.0" metadata-complete="true"
+>
+    <name>externalstore_plugin</name>
+
+    <!--
```

Comment:
The servlet should be enabled by default. In the init method of the Servlet it should check the configuration and if it should not be used you can throw a UnavailableException, which lets the ServletContainer disable the Servlet.